h2o-genmodel/src/main/java/hex/genmodel/algos/isoforextended/ExtendedIsolationForestMojoModel.java

```diff
@@ -0,0 +1,26 @@
+package hex.genmodel.algos.isoforextended;
+
+import hex.genmodel.algos.tree.SharedTreeMojoModel;
+
+public final class ExtendedIsolationForestMojoModel extends SharedTreeMojoModel {
+
+  public ExtendedIsolationForestMojoModel(String[] columns, String[][] domains, String responseColumn) {
+    super(columns, domains, responseColumn);
+  }
+
+  @Override
+  public double[] unifyPreds(double[] row, double offset, double[] preds) {
+    return new double[0];
```

Comment:
Should be correctly implemented.

Reply:
It is not used in the current implementation of Extended Isolation Forest

Reply:
So then you should return `H2O.unimpl()`, or something like that.

Reply:
The same question here, @michalkurka, what is your opinion? :)

Reply:
@michalkurka Maybe the purpose of this method is convert anomaly_score into Anomaly/NotAnomaly prediction?

Reply:
Mojo is postponed.